src/Commands/SSHKey/AddCommand.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * @file
- * Contains Pantheon\Terminus\Commands\Auth\SSHKey\AddCommand
+ * Contains Pantheon\Terminus\Commands\SSHKey\AddCommand
```

Comment:
@ronan These are wanted, I'm just mulling over where I want them placed - here under `<?php` or right above the class definition. We're putting them in different places. Do you have an opinion?